packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/fork.html

```diff
@@ -1,14 +1,630 @@
+                    data-name="radio-group-restore-mode"
+                    data-value="$ctrl.RESTORE_MODES.TIMESTAMP"
+                    data-inline
+                >
+                    <span
+                        data-ng-bind=":: ('pci_databases_backups_fork_spec_restore_mode_' + $ctrl.RESTORE_MODES.TIMESTAMP) | translate"
+                    >
+                    </span>
+                </oui-radio>
+                <oui-radio
+                    data-disabled="$ctrl.backupList.length === 0"
+                    data-model="$ctrl.model.restoreMode"
+                    data-name="radio-group-restore-mode"
+                    data-value="$ctrl.RESTORE_MODES.BACKUP"
+                    data-inline
+                >
+                    <span
+                        data-ng-bind=":: ('pci_databases_backups_fork_spec_restore_mode_' + $ctrl.RESTORE_MODES.BACKUP) | translate"
+                    >
+                    </span>
+                </oui-radio>
+            </div>
+            <!-- input data source-->
+            <oui-field
+                data-label="Backup"
+                data-ng-if="$ctrl.RESTORE_MODES.BACKUP === $ctrl.model.restoreMode"
+            >
+                <label class="oui-select">
+                    <select
+                        name="backup"
+                        data-ng-model="$ctrl.model.backupId"
+                        class="oui-select__input"
+                    >
+                        <option
+                            data-ng-repeat="backup in $ctrl.backupList"
+                            value="{{backup.id}}"
+                            data-ng-bind="(backup.createdAt | date: 'medium') + ' (' + (backup.size.value | bytes:2:false:'B') + ')'"
+                        >
+                        </option>
+                    </select>
+                </label>
+            </oui-field>
+            <oui-field
+                data-label="Timestamp"
+                data-ng-if="$ctrl.RESTORE_MODES.TIMESTAMP === $ctrl.model.restoreMode"
+            >
+                <oui-calendar
+                    data-model="$ctrl.model.timestamp"
+                    data-format="Z"
+                    data-alt-format="j M Y H:i:s"
+                    data-enable-time
+                    data-options="$ctrl.calendarOptions"
+                >
+                </oui-calendar>
+            </oui-field>
+        </section>
+        <section id="plan" class="mt-4">
+            <h2 data-translate="pci_databases_backups_fork_plan_label"></h2>
+            <database-plans-list
+                data-current-plan="$ctrl.originalPlan"
+                data-plans="$ctrl.plans"
+                data-selected-plan="$ctrl.model.plan"
+                data-allow-lower-selection="false"
+                data-user="$ctrl.user"
+                data-show-monthly-prices="$ctrl.showMonthlyPrices"
+                data-on-change="$ctrl.onPlanChanged(selectedPlan)"
+            ></database-plans-list>
```

Comment:
**nitpick:** Component can be renamed to have prefic `pci-project-database-plans-list`

Reply:
This will be done later with a refactoring PR to handle this cleanly, as we'd like to avoid asking for quality check on pages that are using this component, but not in the scope of this feature.